tests/integration_tests/test_hyperopt.py

```diff
@@ -557,3 +557,34 @@ def test_hyperopt_nested_parameters(csv_filename, tmpdir, ray_cluster_7cpu):
             assert trial_config[TRAINER]["learning_rate_scaling"] == "linear"
 
         assert trial_config[TRAINER]["learning_rate"] in {0.7, 0.42}
+
+
+def test_hyperopt_without_config_defaults(csv_filename, tmpdir, ray_cluster_7cpu):
+    input_features = [category_feature(encoder={"vocab_size": 3})]
+    output_features = [category_feature(decoder={"vocab_size": 3})]
+
+    rel_path = generate_data(input_features, output_features, csv_filename)
+
+    config = {
+        INPUT_FEATURES: input_features,
+        OUTPUT_FEATURES: output_features,
+        COMBINER: {TYPE: "concat"},
+        TRAINER: {"train_steps": 5, "learning_rate": 0.001, BATCH_SIZE: 128},
+        # Missing search_alg and executor, but should still work
+        HYPEROPT: {
+            "parameters": {
+                "trainer.learning_rate": {
+                    "lower": 0.0001,
+                    "upper": 0.01,
+                    "space": "loguniform",
+                }
+            },
+            "goal": "minimize",
+            "output_feature": output_features[0]["name"],
+            "validation_metrics": "loss",
```

Comment:
I've also seen `metric` being used in some hyperopt configs (e.g. https://ludwig.ai/latest/configuration/hyperparameter_optimization/). Which one is the official/most recent parameter name?

Reply:
Interesting. @arnavgarg1 note that `validation_metrics` is not currently on the hyperopt schema.

Reply:
Good catch. The reason the test still works is because of backward compatibility. In either case, I've changed this to be `metric` because that is the latest. Apologies for the confusion (copied the config from another test and then modified it)